site/website/javadoc.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find ../javadoc -name "*.tar.gz" -exec tar xzvf {} -Cstatic/javadoc \;
```

Comment:
what about to add `mkdir -p static/javadoc` before this line. Currently tt will fail if `static/javadoc` dir does not exist.

Reply:
hm, actually `mkdir -p` will rewrite it. We should rather check whether dir is existed and create it only if not.